pkg/parser/model/model.go

```diff
@@ -541,6 +541,8 @@ type TableInfo struct {
 	ExchangePartitionInfo *ExchangePartitionInfo `json:"exchange_partition_info"`
 
 	TTLInfo *TTLInfo `json:"ttl_info"`
+
+	DBID int64 `json:"-"`
```

Comment:
It's better to add some tests to make sure some DDL like `rename table` will reset table's `DBID` correctly, especially for schema diff load

Reply:
done